doc/developer/blue-app-monero.rst

```diff
@@ -408,7 +407,7 @@ to indicate another command will be sent.
 +---------------+----------------------------------------------------------+
 | ``x--------`` | Last sub command indicator                               |
 |               |                                                          |
-| ``1--------`` | More identical subcommand to comme                       |
```

Comment:
Not sure what was meant by "comme" here, but I assumed it was supposed to be "command".  Please let me know if this is incorrect.

Reply:
I think it's meant as "to come", so "More identical subcommand to come". Perhaps "More identical subcommand forthcoming" would be better.